### ISSUE TITLE
Temp files created by CLR Profiler are not deleted sometimes

### ISSUE THREAD

**Issue by JuntaoZhu** [Azure/DaaS#69]
`Uncompressing C:\local\Temp\Logs\xxxxxxx\xxxxxxxxxxxxx\xxxxxxxxxxx.diagsession to C:\local\Temp\ETL.xxxxxxxxxxxxxx...
`
Failed to extract files due to this exception:
```
Failed while extracting ETL Trace. LocalFilePath is C:\local\Temp\Logs\xxxxxxxxxxxxxxxxxx_w3wp_5144_20220718-190155.diagsession_sc.user_aux.etl and Disk Usage Details are  C:\local has totalBytes = [150323855360]  and freeBytes = [125559492608] and UsedPercent = 16 , C:\home has totalBytes = [268435456000]  and freeBytes = [266105233408] and UsedPercent = 1  and Exception System.Runtime.InteropServices.COMException:There is not enough space on the disk. (Exception from HRESULT: 0x80070070)

    at DiagnosticsHub.Packaging.Interop.IDhPackage.ExtractResourceToPath(Guid& ResourceId, String extractionPath)

   at Microsoft.DiagnosticsHub.Packaging.InteropEx.DhPackageNative.ExtractResourceToPath(Guid& resourceId, String extractionPath)

   at ClrProfilingAnalyzer.Program.GetLocalFilePath(String packageFilePath, DhPackage package, ResourceInfo resource, String fileExtension)
```

**Comment by JuntaoZhu** [comment]
@puneet-gupta Could you share the update of the fix in this Issue? Thank you.

**Comment by puneetg1983** [comment]
This is fixed with https://github.com/Azure/DaaS/pull/71 

The fix will roll out with DaaS 3.1 deployment which should start by next week.

Closing this issue.
